resource/menu/client/cl_base.lua

```diff
@@ -10,10 +10,20 @@ isMenuDebug = false
 isMenuVisible = false
 menuPermissions = {}
 lastTpCoords = false;
+RedM = false
 local isMenuEnabled = (GetConvar('txAdmin-menuEnabled', 'false') == 'true')
 
+--Redm support
+CreateThread(function()
+  Wait(2000)
+  if (GetGameName() == "redm") then
+    RedM = true
+  else
+    RedM = false
+  end
+end)
```

Comment:
Correct me if I'm wrong, as I cannot test this within RedM atm, but `GetGameName()` should be available at init script eval; any reason this thread needs to be here?

Also small nit, but it would be appreciated if the variable name was along the lines of `IsGameRedM`

```suggestion
IsGameRedM = GetGameName() == "redm"
local isMenuEnabled = (GetConvar('txAdmin-menuEnabled', 'false') == 'true')
```

Reply:
there are many ways to do code , but sure  thats way better looking too.